quickstart/monitoring/monitoring-quickstart.ipynb

```diff
@@ -170,7 +169,8 @@
     "prod_data = pd.read_parquet(\"https://quickstart-sample-data.s3.us-west-2.amazonaws.com/diabetes_readmission_v2/prod.parquet\")\n",
     "\n",
     "# If __id__ in the data, remove this. \n",
-    "prod_data.rename(columns={'__id__': _ID_COLUMN_NAME}, inplace=True)"
+    "prod_data.rename(columns={'__id__': _ID_COLUMN_NAME}, inplace=True)\n",
+    "prod_data[\"prediction_str\"] = prod_data[\"prediction\"].apply(lambda x: \"true\" if float(x) > .5 else \"false\")"
```

Comment:
We should use the threshold value from variable: `_PREDICTION_THRESHOLD `?

Reply:
We should replace above line with this? 
```
prod_data[_PREDICTION_CLASSIFICATION_COLUMN_NAME] = prod_data[_PREDICTION_COLUMN_NAME].apply(lambda x: "1" if float(x) > _PREDICTION_THRESHOLD else "0")
```

Reply:
Also, we can define `_PREDICTION_CLASSIFICATION_COLUMN_NAME` variable along with other variables and remove `add_prediction_classification_column(prod_data, _PREDICTION_THRESHOLD)` and related code since we're adding the `prediction_str` column here itself. @maximilian-reinsel